app/assets/javascripts/discourse/app/components/user-menu/menu.js

```diff
@@ -164,6 +183,33 @@ const CORE_BOTTOM_TABS = [
   },
 ];
 
+const CORE_OTHER_NOTIFICATIONS_TAB = class extends UserMenuTab {
+  @tracked groupedUnreadNotifications =
+    this.currentUser.grouped_unread_notifications;
+
+  get id() {
+    return "other";
+  }
+
+  get icon() {
+    return "discourse-other-tab";
+  }
+
+  get panelComponent() {
+    return "user-menu/other-notifications-list";
+  }
+
+  get count() {
+    return this.site.unassignedNotificationTypes.reduce(
+      (sum, notificationType) => {
+        const key = this.site.notification_types?.[notificationType];
+        return sum + (this.groupedUnreadNotifications?.[key] || 0);
```

Comment:
(continuation from my other comment)

What you want to do here is to use the `this.currentUser.grouped_unread_notifications` directly and use `get()` to retrieve the count of each type like so:

```js
this.currentUser.get(`grouped_unread_notifications.${type}`)
```

That way, Ember will know that the `grouped_unread_notifications.${type}` property needs to be tracked and re-render the UI if it changes. The `this.getUnreadCountForType()` method exists for this very reason (see the comment) so I'd recommend that you use it:

https://github.com/discourse/discourse/blob/de8cd1943844e224d88e427a4554d0d4f323e885/app/assets/javascripts/discourse/app/lib/user-menu/tab.js#L46-L59